Makefile.kokkos

```diff
@@ -13,7 +13,7 @@ KOKKOS_DEVICES ?= "Threads"
 # NVIDIA:   Kepler,Kepler30,Kepler32,Kepler35,Kepler37,Maxwell,Maxwell50,Maxwell52,Maxwell53,Pascal60,Pascal61,Volta70,Volta72,Turing75,Ampere80,Ampere86,Hopper90
 # ARM:      ARMv80,ARMv81,ARMv8-ThunderX,ARMv8-TX2,A64FX
 # IBM:      BGQ,Power7,Power8,Power9
-# AMD-GPUS: Vega900,Vega906,Vega908,Vega90A
+# AMD-GPUS: Vega906,Vega908,Vega90A,NAVI1030
```

Comment:
Any particular reason why to spell it `NAVI1030` instead of `Navi1030`?
I am asking because, why I don't necessary love how we have capitalized things in the past, this change would introduce more inconsistency.